### PR TITLE
Split Article Python Functions

### PR DESCRIPTION
**Pull request by Rafase282**
Splits the article into smaller parts.

### CODE REVIEW

**Comment by theflametrooper** [comment]
@alayek please check this :P

**Comment by Rafase282** [comment]
I found an error on the linking back, it should be `Python-Functions` not `Python-Function` Will fix that now.

File: Python-Functions-Defining.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```python
+>>> def fib(n):    # write Fibonacci series up to n
+...     """Print a Fibonacci series up to n."""
+...     a, b = 0, 1
+...     while a < n:
+...         print(a, end=' ')
+...         a, b = b, a+b
+...     print()
+...
```
Comment (alayek) [inline Python-Functions-Defining.md]: I think here we need a REPL snippet. Maybe not in this PR though.

**Comment by alayek** [comment]
# LGTM :+1: 

@koustuvsinha take a look. Hope we are not going too granular!

**Comment by koustuvsinha** [comment]
# LGTM too :+1:

**Pull request merged**